docs/customize/configure.md

```diff
@@ -0,0 +1,621 @@
+# Configuration Variables
+
+This document provides a comprehensive list and explanation of the configuration variables available in the `invenio-app-ils` backend. These variables can be set in the configuration file(s) to customize the behavior of your Invenio ILS application.
```

Comment:
Same remark/comment as in the other PR. I would stick to `InvenioILS` (one word) and avoid modules name.
Or, if you want to explicit mention `invenio-app-ils`, then I would explain that is the module (and not really the backend in the true sense of the term) with a link:
```suggestion
This document provides a comprehensive list and explanation of the configuration variables available in the [invenio-app-ils](https://github.com/inveniosoftware/invenio-app-ils) module. These variables can be set in the configuration file(s) to customize the behavior of your InvenioILS application.
```